test/multi-runtime-services.test.ts

```diff
@@ -0,0 +1,103 @@
+import * as assert from 'assert';
+import * as path from 'path';
+import { cleanOpenRuntimes, launchAndConnect } from './multi-runtime-utils';
+import { delayPromise } from './delay-promise';
+import * as sinon from 'sinon';
+import * as fs from 'fs';
+
+describe ('Multi Runtime Services', () => {
+    beforeEach(async () => {
+        await cleanOpenRuntimes();
+    });
+
+    after(async () => {
+        await cleanOpenRuntimes();
+    });
+
+    // tslint:disable-next-line
+    describe('Multi Runtime with External Provider', function () {
+
+        it('Should work in multi runtime with an external Provider', function(done: any) {
+            // tslint:disable-next-line no-invalid-this
+            this.timeout(8000);
+            const appConfig = JSON.parse(fs.readFileSync(path.resolve('test/app.json')).toString());
+            const url = appConfig.startup_app.url;
+            const newUrl = url.slice(0, url.lastIndexOf('/')) + '/client.html';
+
+            const clientConfig = {
+                'name': 'service-client-test',
+                'url': newUrl,
+                'uuid': 'service-client-test',
+                'autoShow': true,
+                'saveWindowState': false,
+                'nonPersistent': true,
+                'experimental': {
+                    'v2Api': true
+                }
+            };
+
+            async function test () {
+                const spy = sinon.spy();
+                const [fin, finA] = await Promise.all([launchAndConnect(), launchAndConnect()]);
+                const provider = await finA.Service.register();
+                provider.register('test', () => {
+                    spy();
+                    return 'return-test';
+                });
+                provider.onConnection(c => {
+                    spy();
+                });
+                const client = await fin.Application.create(clientConfig);
+                await client.run();
+                await delayPromise(1000);
```

Comment:
you could use the DELAY_MS constant in multi-runtime-utils. Will make changing easier.